src/NetscriptFunctions/Singularity.ts

```diff
@@ -738,6 +738,7 @@ export function NetscriptSingularity(
       helper.checkSingularityAccess("stopAction");
       if (player.isWorking) {
```

Comment:
I suspect because we are working something else might need to be done.  @hydroflame thoughts? I see there is a singularityStopWork function.